From 4c9ab89e36a4d6d1109431bacd0474b5fd22f325 Mon Sep 17 00:00:00 2001
From: Albin B <albin.bala.krishnan@intel.com>
Date: Fri, 4 Dec 2015 14:17:15 +0530
Subject: [PATCH 2/4] [EM/PMIC_CCSM]: Do not check Vbus detection for sink
 event

As we are receiving sink event form the detect, VBUS will be present
by defult. Hence removing the additional vbus status checking in sink
event handling path.

Change-Id: I89660a62fc3021ba9be0c9f2f8e23c6647c39837
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-12471
Signed-off-by: Albin B <albin.bala.krishnan@intel.com>
Reviewed-on: https://android.intel.com:443/444403
(cherry picked from commit 883d0f6708e9428a16d10b38b56ff30e0331b717)
Reviewed-on: https://android.intel.com:443/447277
---
 drivers/power/intel_pmic_ccsm.c | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/drivers/power/intel_pmic_ccsm.c b/drivers/power/intel_pmic_ccsm.c
index 337b64c..588ca8f 100644
--- a/drivers/power/intel_pmic_ccsm.c
+++ b/drivers/power/intel_pmic_ccsm.c
@@ -1844,23 +1844,6 @@ static void pmic_ccsm_process_cable_events(enum cable_type cbl_type,
 		notify_otg = true;
 		break;
 	case CABLE_TYPE_SINK:
-		/* Check for VBUS status */
-		if (cable_state) {
-			ret = pmic_read_reg(chc.reg_map->pmic_schgrirq1, &val);
-			if (ret < 0) {
-				dev_err(chc.dev,
-					"%s: Failed to read VBUS status\n",
-					__func__);
-				goto vbus_fail;
-			}
-			if (!(val & CHRGRIRQ1_SVBUSDET_MASK)) {
-				dev_err(chc.dev,
-				"%s: VBUS not preset in sink mode, val=%x\n",
-					__func__, val);
-				goto vbus_fail;
-			}
-			dev_info(chc.dev, "%s: VBUS Present\n", __func__);
-		}
 		/* Do charger detection and send notification
 		 * to power sypply framework.
 		 */
-- 
1.9.1

